repos/spack_repo/builtin/packages/root/package.py

```diff
@@ -395,6 +435,7 @@ class Root(CMakePackage):
 
     # Optional dependencies
     depends_on("arrow", when="+arrow")
+    depends_on("civetweb", when="+http")
```

Comment:
```suggestion
    depends_on("civetweb@1.17: +shared", when="+http")
```

Reply:
Since the upgrade of civetweb in the latest ROOT release was not due to any particular feature (just general upgrade), I don't think there's really a feature after 1.16 that is required. So, I'd remove this version limit here (in particular since it's not an upstream release) and go with `+shared` only: 
```suggestion
    depends_on("civetweb +shared", when="+http")
```